ooiservices/requirements/common.txt

```diff
@@ -45,5 +45,4 @@ Flask-Cors==1.10.3
 seawater==3.3.2
 simplejson==3.6.5
 SQLAlchemy-Searchable==0.9.2
-uwsgi==2.0.11.1
```

Comment:
@oceanzus , this causes issues when installing locally - can we remove it here and maybe put in a deployment file?